web/web-itests/pom.xml

```diff
@@ -47,13 +47,12 @@
         <logback.version>1.5.16</logback.version>
         <org.apache.aries.blueprint.sample.version>1.0.0</org.apache.aries.blueprint.sample.version>
         <org.apache.aries.blueprint.version>1.0.0</org.apache.aries.blueprint.version>
-        <org.apache.aries.proxy.version>1.0.1</org.apache.aries.proxy.version>
+        <org.apache.aries.proxy.version>1.1.14</org.apache.aries.proxy.version>
```

Comment:
Not sure this proxy release fully supports JDK 17+.

Reply:
Definitely not
there is a problem with sun classes even in java 11+
You can take a look at #414

Reply:
web does not use proxy at all so I removed the dependency together with unused blueprint dependencies